vignettes/databases.Rmd

```diff
@@ -0,0 +1,136 @@
+---
+title: "Use External Databases with shiny.telemetry"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Use External Databases with shiny.telemetry}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+The `{shiny.telemetry}` package can be used with any Shiny application and in this guide we will show how to use it with different databases backend.
+
+A requirement for this guide is to have a database server running and the necessary permissions to create a database, tables and insert data.
```

Comment:
Maybe expand this a little bit? Creating DB and tables can be done e.g. by an infra team. Moreover, it makes sense for the account used to push telemetry data to the DB to have only limited permissions (only to insert the data).

Reply:
Done in d80f0a8, WDYT?